# HG changeset patch
# User Rik <rik@octave.org>
# Date 1486661475 28800
#      Thu Feb 09 09:31:15 2017 -0800
# Branch stable
# Node ID 039dfdc2d2ada3f24bdfa9154e848c98bf8fedb2
# Parent  6e7f8a92a8521f6db45aa1370a7193b845f6875d
acinclude.m4: Correct test for SetPlaceHolderText.

* acinclude.m4: #include <qglobal.h> rather than <Qt/qglobal.h> which may not
exist.

diff --git a/m4/acinclude.m4 b/m4/acinclude.m4
--- a/m4/acinclude.m4
+++ b/m4/acinclude.m4
@@ -489,17 +489,17 @@ AC_DEFUN([OCTAVE_CHECK_FUNC_SETPLACEHOLD
   AC_CACHE_CHECK([whether Qt has SetPlaceholderText function],
     [octave_cv_func_setplaceholdertext],
     [AC_LANG_PUSH(C++)
     ac_octave_save_CPPFLAGS="$CPPFLAGS"
     ac_octave_save_CXXFLAGS="$CXXFLAGS"
     CPPFLAGS="$QT_CPPFLAGS $CXXPICFLAG $CPPFLAGS"
     CXXFLAGS="$CXXPICFLAG $CXXFLAGS"
     AC_PREPROC_IFELSE([AC_LANG_PROGRAM([[
-        #include <Qt/qglobal.h>
+        #include <qglobal.h>
         ]], [[
         #if QT_VERSION < 0x040700
         #error No SetPlacholderText function available.
         #endif
         ]])],
       octave_cv_func_setplaceholdertext=yes,
       octave_cv_func_setplaceholdertext=no)
     CPPFLAGS="$ac_octave_save_CPPFLAGS"
